src/sql/parts/dashboard/containers/dashboardGridContainer.component.ts

```diff
@@ -105,15 +106,15 @@ export class DashboardGridContainer extends DashboardTab implements OnDestroy, O
 		let content = this.getContent(row, col);
 		let colspan: string = '1';
 		if (content && content.colspan) {
-			colspan = content.colspan;
+			colspan = this.covertToNumber(content.colspan, this.cols.length).toString();
```

Comment:
typo: `convertToNumber` .  Also, presumably convertToNumber won't ever return `undefined` based on how it's being used, right?

Reply:
it returns 1 by default

---
In reply to: [172673606](https://github.com/Microsoft/sqlopsstudio/pull/847#discussion_r172673606) [](ancestors = 172673606)